cumulusci/robotframework/Salesforce.py

```diff
@@ -81,6 +82,21 @@ def builtin(self):
     def cumulusci(self):
         return self.builtin.get_library_instance("cumulusci.robotframework.CumulusCI")
 
+    def initialize_location_strategies(self):
+        """Initialize the Salesforce location strategies 'text' and 'title'
+        plus any strategies registered by other keyword libraries
+
+        Note: This keyword is called automatically from Open Test Browser
+        """
+        locator_manager.register_locators("cci", lex_locators)
```

Comment:
Possibly we should use `sf` as the prefix for the core set of Salesforce locators -- i.e. name them after the functional domain that they're related to, rather than the fact that they happen to be defined by CumulusCI.

Reply:
After letting Jason weigh in, I'll rename it to `sf`. Thanks for the suggestion!